selenium.go

```diff
@@ -411,6 +411,8 @@ type WebElement interface {
 	IsDisplayed() (bool, error)
 	// GetAttribute returns the named attribute of the element.
 	GetAttribute(name string) (string, error)
+	// GetProperty returns the named attribute of the element.
```

Comment:
Can you add a comment about how this is different from GetAttribute or otherwise differentiate between the comment prose of the two methods?

Reply:
I extended the function descriptions.
It is not easy topic, because the older Chrome version did not distinguish between them and returned the DOM properties for the GetAttribute method.